releasenotes/notes/0.3/prepare-0.3-d9049766293a3be7.yaml

```diff
@@ -0,0 +1,9 @@
+---
+prelude: |
```

Comment:
Feels like wire cutting and general 2-qubit unitary gate-cutting support is worth a sentence here? Maybe not?

Reply:
Yeah, I think this makes sense.  I initially decided against it, because the arbitrary gates thing ends up listed immediately after the prelude.  But wire cutting is the last listed feature.  I think it looks great with the additional sentence, added in 82ada2b8c5b7392c8365d510b51df19fcb7cac4c.